examples/html-styled/elm.json

```diff
@@ -4,29 +4,29 @@
         ".",
         "../../src"
     ],
-    "elm-version": "0.19.0",
+    "elm-version": "0.19.1",
```

Comment:
I assume this can be 0.19.0 too, yeah? 😄

Reply:
:smile: Indeed! Reverted!